ts/src/renderer/phaser/classes/PhaserRegion.ts

```diff
@@ -13,11 +13,7 @@ class PhaserRegion extends Phaser.GameObjects.Graphics {
 
 		scene.add.existing(this);
 
-		this.transformListener = region.on('transform', () => {
-
-			this.transform();
-
-		});
+		this.transformListener = region.on('transform', this.transform, this, false); // hack for now
```

Comment:
we need fix for `on` method before merging this in

Reply:
done :D